homeworks/dmytro.shevchenko_dmytro-shevchenko1/4-basics/basics.js

```diff
@@ -0,0 +1,25 @@
+const firstNumber = 77;
+const secondNumber = 33;
+
+const addition = firstNumber + secondNumber;
+const subtraction = firstNumber - secondNumber;
+const multiplication = firstNumber * secondNumber;
+const division = firstNumber / secondNumber;
+const remainder = firstNumber % secondNumber;
+const exponentiation = (firstNumber ** secondNumber);
+
+document.getElementById('output').innerHTML = `
```

Comment:
It is considered a best practice to render page content using HTML rather than relying solely on JavaScript. HTML loads faster, allowing users to see some content while the page is loading. Please move the HTML content to the index.html file

Reply:
Get HTML element by data-attributes, like `data-output`.
Don't forget to add `data-output` to HTML tag